pom.xml

```diff
@@ -5,9 +5,14 @@
 
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-spanner-hibernate</artifactId>
-  <packaging>jar</packaging>
+  <packaging>pom</packaging>
   <version>0.1.0-SNAPSHOT</version>
 
+  <modules>
+    <module>google-cloud-spanner-hibernate-dialect</module>
+    <module>google-cloud-spanner-hibernate-testing</module>
+  </modules>
+
```

Comment:
Below, you have the `dependencies` section. Parent pom should not have any dependencies, only `dependencyManagement` to control the versions.

Reply:
Done. I kept the test-related deps in here though because they are shared by all modules and this is how spring-cloud-gcp does it.